### DISCUSSION TITLE
No review for the pull request

### DISCUSSION THREAD

**Discussion by Saipradyumnagoud** [sanjay-kv/Open-source-Practice#1018] [Q&A] [answered]
https://github.com/Omkar-Sonawane-23/Maa-Vimala/pull/318

this is the pull request

**Comment by sanjay-kv** [comment]
This project has been removed due to inactive

**Comment by AnkitMourya12** [comment, marked as answer]
Yes this is  correct This project has been removed due to inactive. 

but @Saipradyumnagoud  u understand without assigning issue not contributing to project, so as you know  first take permission from mentor or and admin of project then contributing to the project.
